backend/src/main/java/edu/nus/market/controller/AccountController.java

```diff
@@ -59,41 +55,52 @@ public ResponseEntity<Object> register(@Valid @RequestBody Register register, Bi
     }
 
     @DeleteMapping("/me")
-    public ResponseEntity<Object> deleteAccount(@Valid @RequestBody DelAccReq req, BindingResult bindingResult, @RequestHeader("Set-Cookie") String token){
+    public ResponseEntity<Object> deleteAccount(@Valid @RequestBody DelAccReq req, BindingResult bindingResult, @RequestHeader("Cookie") String token){
         if (bindingResult.hasErrors()) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ErrorMsg(ErrorMsgEnum.INVALID_DATA_FORMAT.ErrorMsg));
         }
         //System.out.println(JwtTokenManager.decodeAccessToken(token));
         if (!JwtTokenManager.validateCookie(token)) {
             return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(new ErrorMsg(ErrorMsgEnum.UNAUTHORIZED_ACCESS.ErrorMsg));
         }
-        return accountService.deleteAccountService(req);
+        return accountService.deleteAccountService(req, JwtTokenManager.decodeCookie(token));
     }
 
     @PatchMapping("/me/profile")
-    public ResponseEntity<Object> updateProfile(@Valid @RequestBody UpdateProfileReq req, BindingResult bindingResult, @RequestHeader("Authorization") String token){
+    public ResponseEntity<Object> updateProfile(@Valid @RequestBody UpdateProfileReq req, BindingResult bindingResult, @RequestHeader("Cookie") String token){
         if (bindingResult.hasErrors()) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ErrorMsg(ErrorMsgEnum.INVALID_DATA_FORMAT.ErrorMsg));
         }
         if (!JwtTokenManager.validateCookie(token)) {
             return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(new ErrorMsg(ErrorMsgEnum.UNAUTHORIZED_ACCESS.ErrorMsg));
         }
-        return accountService.updateProfileService(req, Integer.parseInt(JwtTokenManager.decodeCookie(token)));
+        return accountService.updateProfileService(req, JwtTokenManager.decodeCookie(token));
     }
 
-    @PatchMapping("/me/psw")
-    public ResponseEntity<Object> updateAccountPsw(@Valid @RequestBody UpdPswReq req, BindingResult bindingResult, @RequestHeader("Authorization") String token){
+    @PatchMapping("/me/password")
```

Comment:
In REST, if you choose `password` as a resource name, then this should be `PUT /auth/me/password`, because you are updating the whole password resource.
```suggestion
    @PutMapping("/me/password")
```